.github/workflows/ci-test-custom-script.yml

```
@@ -254,11 +254,24 @@ jobs:
+      # - uses: browser-actions/setup-chrome@latest
+      #   id: setup-chrome
+      #   with:
+      #     chrome-version: 129
+      #     install-chromedriver: true
+      #     install-dependencies: true
+      # - run: |
+      #     echo "BROWSER_PATH=${{ steps.setup-chrome.outputs.chrome-path }}" >> $GITHUB_ENV
+      #     ${{ steps.setup-chrome.outputs.chrome-path }} --version
+
+      - name: Install Google Chrome 129.0.6668.100
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:5:47: Double quote to prevent globbing and word splitting

(shellcheck)